### ISSUE TITLE
get rid of writing to /root

### ISSUE THREAD

**Issue by vstavrinov** [htrgouvea/nipe#161]
You are write a lot of data into /root directory. This is very bad style. Please write your data somewhere else by default and make this path configurable.

**Comment by htrgouvea** [comment]
Hi @vstavrinov, are you using the latest version of Nipe? 

The config file of nipe are in ./config, maybe the events in /root are from Tor. Can you provide more informations to me?

Thanks.

**Comment by vstavrinov** [comment]
Sorry, I could not reproduce this problem.  At the first time  there was lot of megabytes in the /root that I have removed as it eat all free space, but now  there is only build.log after repeating installation. But in both cases the result is the same:

`[!] ERROR: sorry, it was not possible to establish a connection to the server.`

So it is good idea of this project, but I don't see the way to get it working.

**Comment by vstavrinov** [comment]
I have found the cause: https://check.torproject.org/api/ip is blocked.  It available via vpn only.  So it  is the vicious circle problem:  this project designed for bypassing blocking, but it itself uses blocked site. More over the problem is deeper. The error above issued by status command.  But I have tried start command with no errors reported. But nothing is working too: neither dns resolution nor bare ip is not accessible.  Also I  repeated this test  with clean iptables and got the same result. What else to do to get it working?
